policyengine_us/parameters/gov/usda/snap/income/deductions/utility/single/phone.yaml

```diff
@@ -10,8 +10,10 @@ metadata:
   uprating: gov.usda.snap.uprating
   propagate_metadata_to_children: true
   reference:
-    - name: USDA utility allowances by state spreadsheet. Extracted by build_utility_allowances()
-      href: https://www.fns.usda.gov/sites/default/files/resource-files/snap-sua-table-fy24-032624.xlsx
+  - name: USDA utility allowances by state spreadsheet. Extracted by build_utility_allowances (FY2024)
+    href: https://www.fns.usda.gov/sites/default/files/resource-files/snap-sua-table-fy24-032624.xlsx
```

Comment:
This link doesn't work for me (the FY 2024 one). Could you fix this and for all other files